src/VSCodeExtension/src/tests/unit/indent.test.ts

```diff
@@ -0,0 +1,51 @@
+import "mocha";
```

Comment:
I'd suggest unit tests to make sure dotted names are preserved as namespace names.